kin-devplatform-sdk/src/main/java/kin/devplatform/Environment.java

```diff
@@ -2,80 +2,98 @@
 
 public class Environment implements KinEnvironment {
 
-	private static Environment PLAYGROUND;
-	private static Environment PRODUCTION;
+    private static Environment PLAYGROUND;
+    private static Environment PRODUCTION;
 
-	private final String blockchainNetworkUrl;
-	private final String blockchainPassphrase;
-	private final String issuer;
-	private final String ecosystemServerUrl;
-	private final String ecosystemWebFront;
-	private final String biUrl;
+    private final String oldBlockchainNetworkUrl;
+    private final String newBlockchainNetworkUrl;
+    private final String oldBlockchainPassphrase;
+    private final String newBlockchainPassphrase;
+    private final String issuer;
+    private final String ecosystemServerUrl;
+    private final String ecosystemWebFront;
+    private final String biUrl;
 
-	public Environment(String blockchainNetworkUrl, String blockchainPassphrase,
-		String issuer, String ecosystemServerUrl, String ecosystemWebFront, String biUrl) {
-		this.blockchainNetworkUrl = blockchainNetworkUrl;
-		this.blockchainPassphrase = blockchainPassphrase;
-		this.issuer = issuer;
-		this.ecosystemServerUrl = ecosystemServerUrl;
-		this.ecosystemWebFront = ecosystemWebFront;
-		this.biUrl = biUrl;
-	}
+    public Environment(String oldBlockchainNetworkUrl, String oldBlockchainPassphrase, String newBlockchainNetworkUrl,
+                       String newBlockchainPassphrase, String issuer, String ecosystemServerUrl, String ecosystemWebFront, String biUrl) {
+        this.oldBlockchainNetworkUrl = oldBlockchainNetworkUrl;
+        this.newBlockchainNetworkUrl = newBlockchainNetworkUrl;
+        this.oldBlockchainPassphrase = oldBlockchainPassphrase;
+        this.newBlockchainPassphrase = newBlockchainPassphrase;
+        this.issuer = issuer;
+        this.ecosystemServerUrl = ecosystemServerUrl;
+        this.ecosystemWebFront = ecosystemWebFront;
+        this.biUrl = biUrl;
+    }
 
-	@Override
-	public String getBlockchainNetworkUrl() {
-		return blockchainNetworkUrl;
-	}
+    @Override
+    public String getOldBlockchainNetworkUrl() {
+        return oldBlockchainNetworkUrl;
+    }
 
-	@Override
-	public String getBlockchainPassphrase() {
-		return blockchainPassphrase;
-	}
+    @Override
+    public String getNewBlockchainNetworkUrl() {
+        return newBlockchainNetworkUrl;
+    }
 
-	@Override
-	public String getIssuer() {
-		return issuer;
-	}
+    @Override
+    public String getOldBlockchainPassphrase() {
```

Comment:
Can we use one method that accepts and IntDef for . `NEW,OLD` ?

Reply:
there's a difference between old and new params, issuer relevant for old only., so you can't generalize all.

Reply:
it still seems overhead to have multiple methods for that.

Reply:
be calm be strong be grateful...
If you really insist we can Objectify it and add all of them to an Object, something like EnvironmentInfo or something like that